src/zui/ZUILockCard.tsx

```diff
@@ -0,0 +1,80 @@
+import { FC } from 'react';
+import { Box, Divider, Switch, Typography, useTheme } from '@mui/material';
+import { Check, Close } from '@mui/icons-material';
+
+import ZUICard from 'zui/ZUICard';
+
+type TipsSection = {
+  bullets: string[];
+  header: string;
+  iconType: 'check' | 'close';
+};
+
+type ZUILockCardProps = {
+  isActive: boolean;
+  lockedHeader: string;
+  lockedSubheader: string;
+  onToggle: (newValue: boolean) => void;
+  tips: {
+    safe: TipsSection;
+    unsafe: TipsSection;
+  };
+  unlockedHeader: string;
+  unlockedSubheader: string;
+};
+
+const ZUILockCard: FC<ZUILockCardProps> = ({
+  isActive,
+  lockedHeader,
+  lockedSubheader,
+  onToggle,
+  tips,
+  unlockedHeader,
+  unlockedSubheader,
+}) => {
+  const theme = useTheme();
+
+  const renderTips = (section: TipsSection) => {
+    const icon =
+      section.iconType === 'check' ? (
+        <Check style={{ color: theme.palette.success.main }} />
```

Comment:
No need to change this, but I think you can just write "color='success'" here, just like you did with "color='error'" below?